src/Domain/LeanCode.DomainModels.EF/EFRepository.cs

```diff
@@ -9,19 +9,20 @@
 
 namespace LeanCode.DomainModels.EF
 {
-    public abstract class EFRepository<TEntity, TIdentity, TContext, TUnitOfWork> : IRepository<TEntity, TIdentity>
+    public abstract class EFRepository<TEntity, TIdentity, TContext> : IRepository<TEntity, TIdentity>
         where TEntity : class, IAggregateRootWithoutOptimisticConcurrency<TIdentity>
         where TIdentity : notnull
         where TContext : notnull, DbContext
-        where TUnitOfWork : notnull, IUnitOfWork
     {
+        private TContext DbContext { get; }
         protected DbSet<TEntity> DbSet { get; }
-        protected TUnitOfWork UnitOfWork { get; }
+        protected Task CommitAsync(CancellationToken cancellationToken = default) =>
```

Comment:
Since this is EF-based implementation, let's stick to `SaveChangesAsync` naming.

Reply:
Should I remove this function if I'll make `DbContext` protected?

Reply:
Or maybe make it public in order to have access to that?

Reply:
Yeah, we can remove that if you find that redundant.

No, I think `public` will be too much. Let's make the final classes decide what they want to do with this.